engine/src/main/java/io/camunda/zeebe/engine/processing/streamprocessor/TypedRecordProcessor.java

```diff
@@ -57,9 +40,7 @@ default void processRecord(
   default void processRecord(
       final long position,
       final TypedRecord<T> record,
-      final LegacyTypedResponseWriter responseWriter,
-      final LegacyTypedStreamWriter streamWriter,
       final Consumer<SideEffectProducer> sideEffect) {
-    processRecord(record, responseWriter, streamWriter, sideEffect);
+    processRecord(record, sideEffect);
   }
```

Comment:
:wrench: We could remove this method from the interface. It is not implemented by any processor.